src/client/cmd/transfer.cpp

```diff
@@ -83,26 +83,31 @@ mp::ReturnCode cmd::CopyFiles::run(mp::ArgParser* parser)
     return dispatch(&RpcMethod::ssh_info, request, on_success, on_failure);
 }
 
-std::string cmd::CopyFiles::name() const
+std::string cmd::Transfer::name() const
 {
-    return "copy-files";
+    return "transfer";
 }
 
-QString cmd::CopyFiles::short_help() const
+std::vector<std::string> cmd::Transfer::aliases() const
 {
-    return QStringLiteral("Copy files between the host and instances");
+    return {name(), "copy-files"};
```

Comment:
I notice this is not reflected in the help, so the alias is hidden. Is that intended?

Reply:
Yes, we've just left it here for muscle-memory purposes.